zad1/main.rb

```diff
@@ -1,13 +1,13 @@
 def sort_letters(string)
-
+	string.chars.sort.join
 end
 
 def count_vowels(string)
-
+	string.count("aeouiy")
 end
 
 def filter_odd(array)
-
+	array.select{|n| n.odd?}
```

Comment:
Tu można, sugerując się prezentacją, skrócić zapis za pomocą shorthandu:

```ruby
array.select &:odd?
```
A tak to super wszystko 👍